### ISSUE TITLE
@glue42/ng compilation error

### ISSUE THREAD

**Issue by swseverance** [Glue42/core#211]
**Describe the bug**
@glue42/ng library appears to have a compilation error

**To Reproduce**
1. Go to `/core/tutorials/angular/start/clients`
2. Run `npm install --save @glue42/ng` as instructed by the angular tutorial. This appears to install version 2.1.8
3. Run `npm start`
4. Observe compilation error:

```
Compiling @glue42/ng : es2015 as esm2015
Error: Failed to compile entry-point @glue42/ng (es2015 as esm2015) due to compilation errors:
node_modules/@glue42/ng/dist/fesm2015/glue42-ng.js:148:17 - error NG2003: No suitable injection token for parameter 'parentModule' of class 'Glue42Ng'.
no type or decorator
```

**Expected behavior**
There should be no compilation errors

**Screenshots**
![image](https://user-images.githubusercontent.com/18009315/117842491-fb19de80-b24b-11eb-8eaa-ad7238171ea0.png)

**Desktop (please complete the following information):**
 - OS: Windows & MacOS
 - Browser: N/a
 - Version: "@glue42/ng": "^2.1.8"

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone X] N/A
 - OS: [e.g. iOS 13] N/A
 - Browser [e.g. stock browser, safari] N/A
 - Version [e.g. 22] N/A

**Additional context**

**Comment by swseverance** [comment]
@flashd2n I compiled the tutorial again using  @glue42/ng  2.1.11 and no longer see the compilation error. You may close this issue if you'd like. Thanks

**Comment by flashd2n** [comment]
@swseverance Sorry I didn't notice your issue before. Yeah, we identified and issues and patched it in @glue42/ng@2.1.11. Closing.
